src/features/vote/helpers/formatVoteDataToCommit.ts

```diff
@@ -47,16 +47,7 @@ export async function formatVoteDataToCommit(
         datum.identifier = el.idenHex;
         // datum.time = el.time;
         datum.time = el.timeBN.toNumber();
-        let ancData = "";
-
-        // anc data is set to - or N/A in UI if empty, convert back to 0x.
-        if (el.ancillaryData === "-" || el.ancillaryData === "N/A") {
-          ancData = "0x";
-        } else {
-          ancData = web3.utils.utf8ToHex(el.ancillaryData);
-        }
```

Comment:
Nice, so we can't assume this conversion works. I'd suggest we wrap this call in a try / catch and set it to your solution -- so we know we aren't breaking anything else.